test/controllers/FileChecksControllerSpec.scala

```diff
@@ -99,9 +100,13 @@ class FileChecksControllerSpec extends FrontEndTestHelper with TableDrivenProper
         val filesProcessedWithChecksum = 12
         val filesProcessedWithFFID = 8
         val dataString: String = progressData(filesProcessedWithAntivirus, filesProcessedWithChecksum, filesProcessedWithFFID, allChecksSucceeded = false)
-
         val uploadStatus = List(ConsignmentStatuses(UUID.randomUUID(), UUID.randomUUID(), UploadType.id, CompletedValue.value, someDateTime, None))
-        mockGetFileCheckProgress(dataString, userType)
+        if (userType == "judgment") {
+          val dataString2: String = progressData(40, 40, 40, allChecksSucceeded = true)
```

Comment:
Can you name the variables so that it easy to know what the numbers mean?
`val dataString2: String = progressData(filesProcessedWithAntivirus = 40, filesProcessedWithChecksum = 40, filesProcessedWithFFID = 40, allChecksSucceeded = true)`